include/deal.II/grid/grid_in.h

```diff
@@ -663,8 +663,26 @@ class GridIn
 
 
   DeclException0(ExcGmshNoCellInformation);
+  
+  /**
+   * Return a map containing field data. The format of the returned map is as follows: 
+   * - std::string stores the name of the field data (identifier) as specified in the external mesh
+   * - std::vector<double > stores value for the given identifier in each cell. To access the value use field_data[name_field][cell_id].
+   */
+  std::map< std::string, std::vector<double> >& get_field_data()
+  {      
+      return field_data;
+  }
 
 protected:
+    
+  /**
+  * Data member that stores cell data. The format is as follows:
+  * - std::string stores the name of the field data (identifier) as specified in the external mesh
+  * - std::vector<double> stores value for the given identifier in each cell_id. To access the value use field_data[name_field][cell_id].
+  */
+  std::map< std::string, std::vector<double> > field_data;
```

Comment:
This field is `protected:`. But I think it could just as well be `private:`, right? If so, please make it `private:`.